keyboards/coseyfannitutti/mullet/keymaps/default/keymap.c

```diff
@@ -0,0 +1,68 @@
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+#include QMK_KEYBOARD_H
+
+#define _BL 0
+#define _FL 1
+
+  /* Qwerty
+   * .---------------------------------------------------------------------------------------------.
+   * | Esc |  1  |  2  |  3  |  4  |  5  |  6  |  7  |  8  |  9  |  0  |  -  |  =  |  Bkspc  | Ins |
+   * |---------------------------------------------------------------------------------------------+
+   * | Tab    |  Q  |  W  |  E  |  R  |  T  |  Y  |  U  |  I  |  O  |  P  |  [  |  ]  |   \  | Del |
+   * |---------------------------------------------------------------------------------------------+
+   * | Caps    |  A  |  S  |  D  |  F  |  G  |  H  |  J  |  K  |  L  |  ;  |  '  |   Enter   | PgUp|
+   * |---------------------------------------------------------------------------------------------+
+   * | Shift     |  Z  |  X  |  C  |  V  |  B  |  N  |  M  |  ,  |  .  |  /  |  Shift  |  U  | Pgdn|
+   * |---------------------------------------------------------------------------------------------+
+   * | Ctrl |  Win  |  Alt  |              Space                | RAlt | FN  |||||  L  |  D  |  R  |
+   * '---------------------------------------------------------------------------------------------'
+   */
+
+  /* FnLayer
+   * .---------------------------------------------------------------------------------------------.
+   * | ` ~ |  F1 |  F2 |  F3 |  F4 |  F5 |  F6 |  F7 |  F8 |  F9 | F10 | F11 | F12 |  DELETE |PNTSC|
+   * |---------------------------------------------------------------------------------------------+
+   * | Tab    |STATC|BRTHE|RNBOW|RESET|     |     |     |     |     |     |PAUSE|     |  \   |     |
+   * |---------------------------------------------------------------------------------------------+
+   * | Caps    |RGBH+|RGBS+|RGBB+|     |     |     |     |     |     |     | INS |    Enter  | HOME|
+   * |---------------------------------------------------------------------------------------------+
+   * | Shift     |RGBH-|RGBS-|RGBB-|     |     |     |     |RGBM-|RGBM+|RGBTG|  Shift  |VOLUP| END |
+   * |---------------------------------------------------------------------------------------------+
+   * | Ctrl |  Win  |  Alt  |                                   | RAlt | FN  |||||  L  |VOLDN|  R  |
+   * '---------------------------------------------------------------------------------------------'
+   */
+
+const uint16_t PROGMEM keymaps[][MATRIX_ROWS][MATRIX_COLS] = {
+  [_BL] = LAYOUT(
+      KC_GESC, KC_1,    KC_2,    KC_3,    KC_4,   KC_5,   KC_6,   KC_7,   KC_8,   KC_9,    KC_0,    KC_MINS,  KC_EQL,  KC_BSPC,  KC_INS, \
```

Comment:
The backslashes at the end of these lines are not necessary. You can keep them if you want as it isn't hurting anything, but IMO it looks neater and less "hack-y".